src/service/utils/layers.js

```diff
@@ -1,11 +1,13 @@
 import store from '../../store'
 const { commit, state } = store
+const bitsUnitsStorage = ['bps', 'kbps', 'mbps', 'gbps']
 
 export const updateLayers = (evntData) => {
   const { data } = evntData
   const activeQualities = []
   const inactiveQualities = []
-  const encodings = Object.values(data.medias)
+  const mainSource = {'0': data.medias[0]};
```

Comment:
```suggestion
  const mainSource = {'0': data.medias[0]}
```
In general, we prefer not to use semicolons in JS.